src/plugins/api-proxy/util/index.test.js

```diff
@@ -561,71 +560,4 @@ describe('logApiResponse', () => {
 	});
 });
 
-describe('parseRequest', () => {
-	const headers = { authorization: MOCK_AUTH_HEADER };
-	const queries = [mockQuery(MOCK_RENDERPROPS)];
-	it('extracts the queries provided in GET requests', () => {
-		const data = { queries: rison.encode_array(queries) };
-		const getRequest = {
-			headers,
-			method: 'get',
-			query: data,
-			state: {
-				oauth_token: 'foo',
-			},
-			getLanguage: () => 'en-US',
-			server: getServer(),
-		};
-		expect(
-			parseRequest(getRequest, 'http://dummy.api.meetup.com').queries
-		).toEqual(queries);
-	});
-	it('extracts the queries provided in POST requests', () => {
-		const data = { queries: rison.encode_array(queries) };
-		const postRequest = {
-			headers,
-			method: 'post',
-			payload: data,
-			state: {
-				oauth_token: 'foo',
-			},
-			getLanguage: () => 'en-US',
-			server: getServer(),
-		};
-		expect(
-			parseRequest(postRequest, 'http://dummy.api.meetup.com').queries
-		).toEqual(queries);
-	});
-	it('extracts the queries provided in PATCH requests', () => {
-		const data = { queries: rison.encode_array(queries) };
-		const patchRequest = {
-			headers,
-			method: 'patch',
-			payload: data,
-			state: {
-				oauth_token: 'foo',
-			},
-			getLanguage: () => 'en-US',
-			server: getServer(),
-		};
-		expect(
-			parseRequest(patchRequest, 'http://dummy.api.meetup.com').queries
-		).toEqual(queries);
-	});
-	it('throws an error for mal-formed queries', () => {
-		const notAQuery = { foo: 'bar' };
-		const data = { queries: rison.encode_array([notAQuery]) };
-		const getRequest = {
-			headers,
-			method: 'get',
-			query: data,
-			state: {
-				oauth_token: 'foo',
-			},
-			server: getServer(),
-		};
-		expect(() =>
-			parseRequest(getRequest, 'http://dummy.api.meetup.com')
-		).toThrow();
-	});
-});
+describe('getExternalRequestOpts', () => {});
```

Comment:
yeah the commented out import and this stubbed out test were supposed to remind me to finish the test :). Since this function gets separated out into a standalone `send` module in #330, which I'm hoping to merge soon after this initial PR, I think I'll make the testing update there to avoid merge conflicts.